ydb/library/workload/kv_workload.cpp

```diff
@@ -78,6 +78,9 @@ void AddResultSet(const NYdb::TResultSet& resultSet, TVector<TRow>& rows) {
         
         for (size_t col = 0; col < parser.ColumnsCount(); col++) {
             auto& valueParser = parser.ColumnParser(col);
+            if (valueParser.GetKind() == NYdb::TTypeParser::ETypeKind::Optional) {
+                valueParser.OpenOptional();
```

Comment:
We shouldn't have a pair CloseOptional() below?

Reply:
I'm not sure, so ok, let's close it

But it seems to work without because we don't use `valueParser` anymore